mountpoint-s3-crt/src/s3/client.rs

```diff
@@ -829,3 +843,29 @@ pub fn init_default_signing_config(region: &str, credentials_provider: Credentia
 
     SigningConfig(Arc::new(Box::into_pin(signing_config)))
 }
+
+/// The checksum configuration.
+#[derive(Debug, Clone, Default)]
+pub struct ChecksumConfig {
+    /// The struct we can pass into the CRT's functions.
+    inner: aws_s3_checksum_config,
+}
+
+impl ChecksumConfig {
+    /// Create a new [ChecksumConfig] with Crc32c traling checksums.
```

Comment:
```suggestion
    /// Create a new [ChecksumConfig] with Crc32c trailing checksums.
```

Reply:
Also, note that this is only for PUT. It's weird (but understandable) that the CRT combines checksums for PUT and GET into this one struct, but here we are.